src/components/FormContent/FormContent.jsx

```diff
@@ -51,11 +52,13 @@ function FormContent() {
 
       <Grid container justify="flex-end" className={styles['submit-button']}>
         <Button type="submit" variant="contained" color="secondary" size="large">
-          Submit
+          Submit Form
         </Button>
       </Grid>
     </>
   );
 }
 
+FormContent.whyDidYouRender = true;
```

Comment:
OK, I see that you wrapped the whyDidYouRender intialization code in `if (process.env.NODE_ENV === 'development') { ... }` -- so this probably doesn't really hurt anything.  But we could consider also wrapping it here so that none of the `whyDidYouRender` stuff leaks into production code (even if it is just setting a meaningless property at that point).

No strong feelings on this one.  Up to you.

Reply:
I'll just delete that line. ;)